python/cugraph/components/connectivity_wrapper.pyx

```diff
@@ -42,9 +41,39 @@ def weakly_connected_components(graph_ptr, connect_type=CUGRAPH_WEAK):
 
     df = cudf.DataFrame()
     df['labels'] = cudf.Series(np.zeros(num_verts, dtype=np.int32))
-    cdef gdf_column c_labels = get_gdf_column_view(df['labels'])
+    df['vertices'] = cudf.Series(np.zeros(num_verts, dtype=np.int32))
+    
+    cdef cudf_table* tbl = <cudf_table*> table_from_dataframe(df)
```

Comment:
After you are done with `tbl`, you need to `del tbl` corresponding to the `new` in `table_from_dataframe`. Note that this will not actually free any of the columns of the table, so `df` will be unaffected

Reply:
Fixed it! Thanks.